cmdutil/signal_context.go

```diff
@@ -18,7 +17,8 @@ func SignalContext(ctx context.Context, log logrus.FieldLogger) (context.Context
 	ctx, cancel := context.WithCancel(ctx)
 
 	ch := make(chan os.Signal)
-	signal.Notify(ch, []os.Signal{syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}...)
+	ignoredSigs := ignoreSignals()
```

Comment:
I believe those are not ignored signals, but vice versa - the signals we want to listen to

Reply:
yeah those are the signal we are listening to, I think I may have typo it, sorry